### PR TITLE
feat: add docker compose file and running guide

### PR DESCRIPTION
**Pull request by geekyayush**
Here's a demo video

https://github.com/BerriAI/litellm/assets/25874080/9be861fb-7ca3-4933-8203-18edd7e39e25

### CODE REVIEW

**Comment by ghost** [comment]
Hey @geekyayush do you have a linkedin? Would love to shoutout your contribution on the docs.

**Comment by geekyayush** [comment]
> do you have a linkedin?

Yes, here it is: https://www.linkedin.com/in/ayushsomani/
:)

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
lgtm! but can you fix the merge conflicts ?

**Comment by geekyayush** [comment]
> but can you fix the merge conflicts

Sure thing, will get it done.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: "3.9"
+services:
+  litellm:
+    image: ghcr.io/berriai/litellm:main-v1.10.3
```
Comment (PSU3D0) [inline docker-compose.yml]: If this is the default docker-compose it should point to latest

File: docker-compose.yml
Comment (PSU3D0) [inline docker-compose.yml:1]: Maybe rename this to `docker-compose.example.yml`?

Comment (geekyayush) [reply docker-compose.yml:1]: Good idea.
In that case, we can add a notice or something telling people to copy the `docker-compose.example.yml` to `docker-compose.yml`

File: docker-compose.yml (hunk 1 above)
Comment (geekyayush) [reply docker-compose.yml]: Agreed, will update it.
Thanks!

**Comment by geekyayush** [comment]
@krrishdholakia you may check the PR now.

**Comment by ghost** [comment]
great work @geekyayush

**Pull request merged**